utilities_common/multi_asic.py

```diff
@@ -112,6 +112,17 @@ def multi_asic_display_default_option():
 ]
 
 def multi_asic_namespace_validation_callback(ctx, param, value):
+    # TODO: this condition has a bug: it should be either "multi_asic.is_multi_asic()"
+    # or "self.is_multi_asic", but changing it to correct version will lead to the 
+    # following cli not working on single-asic:
+    # show ip(v6) bgp neighbors/network, show starm_control add/del
```

Comment:
What is "show starm_control add/del"? is this existing CLI?

Reply:
sorry for typo, should be "show storm_control"